modules/ROOT/pages/design-best-practices.adoc

```diff
@@ -0,0 +1,117 @@
+= Best Practices when using API Designer
+
+When you perform tasks on Design Center, you may encounter issues. The following topics help you resolve them:
+
+== General Best Practices
+
+* Divide your fragments into shorter reusable dependencies when possible
+
+* Have one version of the same dependency
+
+* Have only one browser tab open at a time for API Designer
+
+=== API Designer Resources
+
+When using API Designer, the resources to parse the specification depend on the user. Having a computer with enough available resources is crucial for API Designer to function properly.
+
+Suggested resources:
```

Comment:
Is this the minimum available or the total minimum?

Reply:
Is "resources available" = resources available to API Designer (above and beyond whatever else the customer is doing)?